lib/noted/notes.ex

```diff
@@ -369,7 +385,15 @@ defmodule Noted.Notes do
         error ->
           error
       end
-    end
+    end)
+    tags
+  else
+    :ok
+  end
+  end
+
+  defp has_notes_tags?(note_id, tag_id) do
```

Comment:
Isn't this an inverse of the logic? I would expect this function to return true if it has notes tags. But it returns true if it returns an empty list.